src/details/ArborX_DetailsMortonCode.hpp

```diff
@@ -114,7 +114,7 @@ unsigned long long morton64(float x, float y)
 {
   // The interval [0,1] is subdivided into 2,147,483,648 bins (in each
   // direction).
-  constexpr unsigned N = (1 << 31);
+  constexpr unsigned N = (1u << 31);
```

Comment:
Revert or add the suffix L98 and L136

Reply:
It is unnecessary in other places. I created a separate PR #640.